site/source/_posts/2018/03/11/Dojo-2-rc1.md

```diff
@@ -5,33 +5,33 @@ authorId: dylan
+As such, if a viable foundational approach exists that we can leverage or extend beyond the core of Dojo 2, we will choose this route rather than creating our own. By leveraging tools like webpack for building and optimizations, various shims for custom elements and Intersection Observers, Globalize and cldr.js for internationalization and more. On top of this foundation, we have built a reliable and consistent out of the box approach making it easy to get started building applications. Dojo 2 does not require each developer to make every architectural choice up front, but still provides easy ways to override our suggested approach when you prefer an alternative.
 
-There are already plenty of [articles that compare frameworks](https://www.sitepen.com/blog/2017/06/13/if-we-chose-our-javascript-framework-like-we-chose-our-music/), but needless to say, we took a long look at everything else because we want Dojo to be part of the ecosystem rather than its silo. By leveraging ES modules, TypeScript interfaces, and custom elements, we provide a robust level of interoperability with other frameworks and libraries.
+There are already plenty of [articles comparing frameworks](https://www.sitepen.com/blog/2017/06/13/if-we-chose-our-javascript-framework-like-we-chose-our-music/), but needless to say, we took a long look at everything else because we want Dojo to be part of the ecosystem rather than its own silo. By leveraging ES modules, TypeScript interfaces, and custom elements, we provide a robust level of interoperability with other frameworks and libraries.
 
-Our focus on developer ergonomics is to get thousands of small details to feel just right. For example, instead of manually worrying about code splitting, we handle this optimization out of the box with zero configuration. Or when you want to style a component, the list of class names available to that component is type safe, saving time in looking up the relevant class names from your CSS file.
+Our focus on developer ergonomics is to get thousands of small details to feel just right. For example, instead of manually worrying about code splitting, we handle this optimization out of the box with zero configuration. Or when you want to style a component, the list of class names available to the component is type safe, saving time in looking up the relevant class names from your CSS file.
```

Comment:
I feel that `Code optimization out of the box` should be a headline here with details underneath. As well as `Css-modules as standard` with a blurb about how localisation fixes style leakage and uses post css to deliver css-next syntax etc... etc...

Reply:
@tomdye can you write the blurb that you'd want to see here?

Reply:
## Css Modules and css-next

Dojo 2 uses css modules out of the box to provide localised class names as standard. This ensures that your styles do not leak out of your widgets and provides type-ahead capability when importing your css files into your widget.

Css-next enables allows Dojo 2 widgets to use cutting edge css syntax and features immediately, such as `calc`, `css-variables` and various colour functions. These are all compiled down to browser friendly variants at build time so there's no need for a post processor such as SCSS or Less.

Reply:
CSS modules ... also a comma after css-variables, and color instead of colour.

Reply:
Added section